tests/test-env-check.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# check for python3
+pytoutput=$(python3 --version 2>&1)
```

Comment:
Seems these rather convoluted checks could be replaced with something like:

```
python3 - << _END_
import sys

if sys.version_info.major < 3 or sys.version_info.minor < 5:
   exit(1)
_END_
```

and then check for $? != 1.